extensions/layout/src/org/sbml/jsbml/ext/layout/Dimensions.java

```diff
@@ -231,21 +232,21 @@ public boolean isIdMandatory() {
    * @return
    */
   public boolean isSetDepth() {
-    return !Double.isNaN(depth);
+    return depth != null && !Double.isNaN(depth);
```

Comment:
Not sure we should test `NaN` here. As it can be value set in purpose if the number is not known.